pricing.js

```diff
@@ -1,37 +1,49 @@
-module.exports.calculateProductPrice = function (product, employee, coverageLevels) {
+function calculateVolLifePrice(product, coverageLevels) {
   var price = 0
   var dollarsOff = 0
+  for (var i = 0; i < coverageLevels.length; i++) {
+    var coverageAmount = coverageLevels[i].coverage
+    price += (coverageAmount / product.cost.costDivisor) * product.cost.price
+  }
 
-  switch (product.type) {
-    case 'volLife':
-      for (var i = 0; i < coverageLevels.length; i++) {
-        var coverageAmount = coverageLevels[i].coverage
+  if (product.employerContribution.mode === 'dollar') {
+    price = price - product.employerContribution.contribution
+  }
+  else {
+    dollarsOff = price * (product.employerContribution.contribution / 100)
+    price = price - dollarsOff
+  }
 
-        price += (coverageAmount / product.cost.costDivisor) * product.cost.price
-      }
+  return parseInt(price * 100) / 100
```

Comment:
Notice that line 17 is repeated on line 34. In the interest of DRY (don't repeat yourself) code you would make this its own function as well.